components/Button/ButtonLink.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { ReactNode } from "react";
+import { CustomLink } from "../CustomLink";
+import { StyledSolidButton } from "./ButtonStyles";
+
+interface Props {
```

Comment:
Use `type` for consistency with other components.

Reply:
i don't get typescript, if you want to make it more TS-friendly then you would have to do it urself 😜